templates/zerver/integrations/git.md

```diff
@@ -0,0 +1,34 @@
+First, download and install our [Python bindings and example scripts](/api) on your Git server.
```

Comment:
Again, this part can be replaced by a macro. You'll have to create one under `templates/zerver/help/include/*`. :)